containers/DvmState.ts

```diff
@@ -81,15 +83,15 @@ const useContractState = () => {
   // get state on setting of contract
   useEffect(() => {
     queryState();
-  }, [dvm, priceIdentifier, expirationTimestamp]);
+  }, [votingContract, priceIdentifier, expirationTimestamp]);
```

Comment:
You should add `storeContract` to this array so that `useEffect()` updates if the store changes

Reply:
done. PTAL